ServerCore/Pages/Events/Details.cshtml

```diff
@@ -133,6 +133,31 @@
         <dd>
             @Model.Event.DefaultLunch
         </dd>
+        <dt>
+            LunchOptions
+        </dt>
+        <dd>
+            @Model.Event.LunchOptions
+            <br />
+            <div style="margin-left:20px;font-size:11px">
+                @{
+                    // Parse lunch options the same way they are in Swag\Register.cshtml and Player\Create or Edit.cshtml
+                    // This will help ensure correct formatting without the need to test those pages as a player
+                    // If the parsing method in those 3 files is ever updated, this will need to be updated as well
+                    var lunchOptions = Model.Event.LunchOptions.Split(";");
+                    foreach (var lunch in lunchOptions)
+                    {
+                        if (!string.IsNullOrWhiteSpace(lunch))
+                        {
+                            string[] lunchParts = lunch.Split(":");
+                            string lunchVal = lunchParts[0].Substring(lunchParts[0].IndexOf("\"") + 1, lunchParts[0].LastIndexOf("\"") - lunchParts[0].IndexOf("\"") - 1);
```

Comment:
Can you please confirm that malformed input doesn't take down the page? Thinking of cases where quotes are missing.

The internet tells me that this might do the trick more safely: `string lunchVal = lunchParts[0].Trim().Trim('"');` - trims whitespace from the edges, then trims quotes from the edges.

Reply:
If this works better, note that the logic would need to be updated in 5 places.

Reply:
Yeah, that's definitely simpler if nothing else, so I like it!